src/main/java/com/shade/decima/ui/data/viewer/texture/controls/ImagePanel.java

```diff
@@ -8,8 +8,35 @@
 import java.awt.event.MouseAdapter;
 import java.awt.event.MouseEvent;
 import java.awt.image.BufferedImage;
+import java.awt.image.FilteredImageSource;
+import java.awt.image.RGBImageFilter;
 import java.util.Objects;
 
+class ChannelFilter extends RGBImageFilter {
```

Comment:
Please don't declare multiple classes in a single file, create a new file instead

Reply:
Should I than place this class in a file in the folder src/main/java/com/shade/decima/ui/data/viewer/texture/util/

Reply:
Yes